rust/agents/validator/src/submit.rs

```diff
@@ -52,12 +53,12 @@ impl ValidatorSubmitter {
         Checkpoint {
             root: tree.root(),
             index: tree.index(),
-            mailbox_address: self.mailbox.address(),
-            mailbox_domain: self.mailbox.domain().id(),
+            merkle_tree_hook_address: self.merkle_tree_hook.address(),
+            mailbox_domain: self.merkle_tree_hook.domain().id(),
```

Comment:
nit: imo now that we renamed `mailbox_address` we should call this `merkle_tree_hook_domain` or something. Or maybe `local_domain`, or `domain`